test/0testSuitCheck.js

```diff
@@ -0,0 +1,28 @@
+const { expect } = require("chai");
+
+describe("Checking test suit state before running unit tests", function () {
+
+  let signer;
+    
+  beforeEach(async function () {
+      
+    // get signers
+    [signer, addr1, addr2, ...addrs] = await ethers.getSigners();
+    
+  });
+
+  it("Ensure mainnet fork is running", async function () {
+        
+    // loading USDT contract	
+    const usdtToken = new ethers.Contract(USDT_ADDRESS, USDT_ABI, signer)  
+    expect(usdtToken.address).to.equal(USDT_ADDRESS);
+
+    // check decimals (USDT has 6 decimals)
+    let decimals = await usdtToken.decimals()
+    expect(decimals).to.equal(6);
+    })
+  });
+
+// USDT mainnet address + ABI
```

Comment:
Should we move these variables to the top near "signer"?

Reply:
Yep. Moved it to a different file